client/index.html

```diff
@@ -62,7 +62,7 @@ <h2>Open Snapdrop on other devices to send files.</h2>
             <use xlink:href="#wifi-tethering" />
         </svg>
         <div id="displayName">The easiest way to transfer data across devices.</div>
-        <div class="font-body2">Allow me to be discovered by: Everyone in this network.</div>
+        <div class="font-body2">This website allows your device to be discovered by everyone on this network.</div>
```

Comment:
I feel like **website** is not the right word here. It's a web app. A more appropriate and general statement would be:

> Your device is discoverable by everyone on this network.